ts/webui/src/components/experiment/trialdetail/chart/DefaultMetricPoint.tsx

```diff
@@ -118,8 +122,32 @@ class DefaultPoint extends React.Component<DefaultPointProps, DefaultPointState>
         };
     }
 
+    private formatAccuracy(accuracy: number | undefined): number {
+        if (Object.is(NaN, NaN) || accuracy === undefined || Object.is(accuracy, Infinity)) {
```

Comment:
`Object.is(NaN, NaN)`?

Reply:
yes, why had this typo? Thanks for your fixing bug online.